app/wallet/accesstracker/accesstracker_test.go

```diff
@@ -0,0 +1,98 @@
+package accesstracker
```

Comment:
I propose just `tracker`, as per Effective Go:

> package name should be good: short, concise, evocative. By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps

Reply:
:+1: i didnt like the name i chose